sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_client_manager.py

```diff
@@ -264,15 +268,119 @@ def __exit__(self, *args):
         self._client.__exit__(*args)
 
 
-class ConfigurationClientManager:
-    def __init__(self, min_backoff_sec: int, max_backoff_sec: int):
-        self._replica_clients: List[ConfigurationClientWrapper] = []
```

Comment:
Does anyone (except `ConfigurationClientManager`) need to access `ConfigurationClientWrapper`? Can we make the class private?

Reply:
We don't expose it as a class, not sure what else we could do? private isn't a thing in python.